run_list_item.go

```diff
@@ -0,0 +1,86 @@
+package chef
+
+import (
+	"fmt"
+	"regexp"
+)
+
+const (
```

Comment:
Why are these constants exported from the package?  They look like they are for internal use only.

Reply:
Updated!